sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ReceiveSingleSessionAsyncSample.java

```diff
@@ -40,14 +40,22 @@ public static void main(String[] args) throws InterruptedException {
             .subscriptionName("<<subscription-name>>")
             .buildAsyncClient();
 
+        // Receiving messages from the first available sessions. It waits up to the AmqpRetryOptions.getTryTimeout().
+        // If no session is available within that operation timeout, it completes with an error. Otherwise, a receiver
+        // is returned when a lock on the session is acquired.
         Mono<ServiceBusReceiverAsyncClient> receiverMono = sessionReceiver.acceptNextSession();
-        Disposable subscription = receiverMono.flatMapMany(receiver -> receiver.receiveMessages()
-            .flatMap(message -> {
-                System.out.println("Processing message from session: " + message.getSessionId());
 
```

Comment:
One of the need for acceptSession() API  was to show user that it can fail. if session does not get accepted they need to do something? Either go to next session or have some code to log message about it.

Reply:
It'll propagate it downstream into the subscriber. So it'll be handled by the onError.